flake.nix

```diff
@@ -30,9 +47,13 @@
         }
       );
 
-      legacyPackages = import ./packages.nix;
+      legacyPackages = lib.warn ''
```

Comment:
I re-build my system with this but now I am getting this warning:


trace: warning: nix-index-database's "legacyPackages" output is deprecated and will be removed
please switch to "packages" instead

Reply:
I am not using any legacyPackages in my configuration:

```
flake.lock
365-    },
366:    "nix-index-database": {
367-      "inputs": {
--
375-        "owner": "nix-community",
376:        "repo": "nix-index-database",
377-        "rev": "bef120c4ea217619579c87862a641ad1c47e2845",
--
382-        "ref": "nixpkgs-fetcher",
383:        "repo": "nix-index-database",
384-        "type": "github"
--
587-        "nix-darwin": "nix-darwin",
588:        "nix-index-database": "nix-index-database",
589-        "nix-ld-rs": "nix-ld-rs",

flake.nix
22-
23:    nix-index-database.url = "github:nix-community/nix-index-database/nixpkgs-fetcher";
24:    nix-index-database.inputs.nixpkgs.follows = "nixpkgs";
25-

home-manager/flake-module.nix
23-                ./common.nix
24:                inputs.nix-index-database.hmModules.nix-index
25:                { programs.nix-index-database.comma.enable = true; }
26-              ];

nixos/flake-module.nix
65-          inputs.nixos-hardware.nixosModules.framework-13th-gen-intel
66:          inputs.nix-index-database.nixosModules.nix-index
67-          inputs.disko.nixosModules.disko
--
72-
73:          { programs.nix-index-database.comma.enable = true; }
74-          #{
```

Reply:
Fixed in https://github.com/nix-community/nix-index-database/pull/106/commits/2408988240f6706906f7b1c6e36424e81e8e3431